lib/FieldType/Mapper/DateTimeFormMapper.php

```diff
@@ -9,45 +9,109 @@
 namespace EzSystems\RepositoryForms\FieldType\Mapper;
 
 use eZ\Publish\Core\FieldType\DateAndTime\Type;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
 use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\DataTransformer\DateTimeValueTransformer;
 use EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface;
+use EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface;
 use EzSystems\RepositoryForms\Form\Type\DateTimeIntervalType;
 use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\DateTimeType;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 
-class DateTimeFormMapper implements FieldDefinitionFormMapperInterface
+/**
+ * FormMapper for ezdatetime FieldType.
+ */
+class DateTimeFormMapper implements FieldDefinitionFormMapperInterface, FieldValueFormMapperInterface
 {
+    /**
+     * @param FormInterface $fieldDefinitionForm
+     * @param FieldDefinitionData $data
+     */
     public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, FieldDefinitionData $data)
     {
         $fieldDefinitionForm
-            ->add('useSeconds', CheckboxType::class, [
-                'required' => false,
-                'property_path' => 'fieldSettings[useSeconds]',
-                'label' => 'field_definition.ezdatetime.use_seconds',
-            ])
-            ->add('defaultType', ChoiceType::class, [
-                'choices' => [
-                    'field_definition.ezdatetime.default_type_empty' => Type::DEFAULT_EMPTY,
-                    'field_definition.ezdatetime.default_type_current' => Type::DEFAULT_CURRENT_DATE,
-                    'field_definition.ezdatetime.default_type_adjusted' => Type::DEFAULT_CURRENT_DATE_ADJUSTED,
-                ],
-                'choices_as_values' => true,
-                'expanded' => true,
-                'required' => true,
-                'property_path' => 'fieldSettings[defaultType]',
-                'label' => 'field_definition.ezdatetime.default_type',
-            ])
-            ->add('dateInterval', DateTimeIntervalType::class, [
-                'required' => false,
-                'property_path' => 'fieldSettings[dateInterval]',
-                'label' => 'field_definition.ezdatetime.date_interval',
-            ]);
+            ->add(
```

Comment:
for next patches, it would be nice to not reindent code you don't really change, this makes big patches and then hard to review patches